main/streams/streams.c

```diff
@@ -2066,7 +2066,7 @@ PHPAPI php_stream *_php_stream_open_wrapper_ex(const char *path, const char *mod
 	}
 
 	if (!path || !*path) {
-		php_error_docref(NULL, E_WARNING, "Filename cannot be empty");
+		zend_value_error("Path cannot be empty");
```

Comment:
I believe we ended up using TypeError for this case everywhere else (because it's often checked by zpp).

Reply:
I thought TypeErrors were just in case the path contained a null byte?

Reply:
I checked again and indeed we use Value error for the empty case: https://heap.space/search?project=php-src&full=%22cannot+be+empty%22&defs=&refs=&path=&hist=&type=c

Reply:
Okay, thanks for checking.